CondTools/SiPixel/test/SiPixelDynamicInefficiencyDB_cfg.py

```diff
@@ -14,7 +14,7 @@
 
 #hptopo
 process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_condDBv2_cff")
```

Comment:
since you are at it, you might as well drop this other obsolete piece: 
https://github.com/cms-sw/cmssw/blob/master/Configuration/StandardSequences/python/FrontierConditions_GlobalTag_condDBv2_cff.py

which just does:

```python
from Configuration.StandardSequences.FrontierConditions_GlobalTag_cff import *
```